src/_includes/tutorial-banner.html

```diff
@@ -0,0 +1,5 @@
+<div class="banner">
```

Comment:
The blue banner is too easy to miss IMHO. Try:

> \<div class="banner alert alert-warning">

(This is what is used here: https://github.com/dart-lang/site-webdev/blob/5-dev/src/_includes/banner.html#L3.)

Reply:
(Btw, this could be a .md file.)